ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -164,19 +165,50 @@ let test_get_nbd_info =
     (* Hosts not connected to either *)
     let _: _ API.Ref.t = make_host sr_of_vdi [] () in
 
+    (* XXX Work is needed here: this fails because get_nbd_info makes XenAPI call host_get_server_certificate *)
```

Comment:
Just a note here for a future PR: We can mock that if we use `Helpers.call_api_functions`, like:
```ocaml
      Helpers.call_api_functions ~__context
        (fun rpc session_id ->
           Client.Client.Event.from ~rpc ~session_id ~classes ~token ~timeout |> Event_types.event_from_of_rpc)
        ~test_fn:(fun () -> Xapi_event.from ~__context ~classes ~token ~timeout |> Event_types.parse_event_from)
```
then we can set `Helpers.test_mode := true` to invoke the mock function `test_fn`. For a complete example, see https://github.com/xapi-project/xen-api/pull/3227

Reply:
But it's fine to do that in a followup PR.